backend/keycloak/entrypoint.sh

```diff
@@ -7,7 +7,7 @@ export KEYCLOAK_PASSWORD="snxTZr9\$5GGnS7w"
 export PROXY_ADDRESS_FORWARDING=true
 
 if [[ -f "/keycloak/realm.json" ]]; then
-    /opt/jboss/tools/docker-entrypoint.sh -b 0.0.0.0 -Djboss.bind.address.private=127.0.0.1 -Djboss.bind.address=0.0.0.0 -Dkeycloak.profile.feature.upload_scripts=enabled -Dkeycloak.migration.action=import -Dkeycloak.migration.provider=singleFile -Dkeycloak.migration.realmName=worksheet -Dkeycloak.migration.file=/keycloak/realm.json
+    /opt/jboss/keycloak/bin/standalone.sh -b 0.0.0.0 -Djboss.bind.address.private=127.0.0.1 -Djboss.bind.address=0.0.0.0 -Dkeycloak.profile.feature.upload_scripts=enabled -Dkeycloak.migration.action=import -Dkeycloak.migration.provider=singleFile -Dkeycloak.migration.realmName=worksheet -Dkeycloak.migration.file=/keycloak/realm.json
```

Comment:
Ezt miért kell átírni?

Reply:
Ahogy behúztuk a mysql is a compose-ba, elkezdett nem indulni a keycloak. Kiderült, hogy a docker-entrypoint scannelheti az alhálózatot és rátalált a mysql-re és úgy döntött, hogy azt akarja használni datasource-ként. A standalone pedig minden esetben H2-t használ és nem foglalkozik a környezettel.